pipeline_games/transform_games.py

```diff
@@ -0,0 +1,81 @@
+"""Script for transforming games data"""
+import pandas as pd
+import numpy as np
+
+
+def identify_unique_tags(data: pd.DataFrame) -> pd.DataFrame:
+    """Generates a tags column from user_tags and genres"""
+    data['tags'] = data['user_tags'].astype(str) + "," + data["genres"]
+    data['tags'] = data['tags'].str.split(',')
+    data = data.explode('tags')
+
+    data = data.drop_duplicates()
+    return data
+
+
+def create_user_generated_column(data: pd.DataFrame) -> pd.DataFrame:
+    """Compares tag to genres and determine if tag is user-generated or not"""
+    data['user_generated'] = np.where(
+        np.isin(data['tags'], data['genres']), False, True)
+
+    return data
+
+
+def drop_unnecessary_columns(data: pd.DataFrame, column_name: str) -> pd.DataFrame:
+    """Removes the no longer need columns"""
+    data = data.drop(column_name, axis=1)
+
+    return data
+
+
+def convert_date_to_datetime(date: str):
```

Comment:
Nit Pick: Type hint for function return

Reply:
so this is a weird type to return but will put it in and if it's too ugly maybe can just go with 'object'?